modules/installation-aws-add-local-zone-locations.adoc

```diff
@@ -36,6 +54,7 @@ Depending on the AWS Region, the list of available zones can be long. The comman
 [source,terminal]
 ----
 $ aws ec2 modify-availability-zone-group \
-    --group-name "<value_of_GroupName>" \
+    --group-name "<value_of_GroupName>" \// <1>
     --opt-in-status opted-in
 ----
+<1> For `<value_of_GroupName>`, specify the name of the group of the Local Zone where you want to create subnets. For example, specify `us-east-1-nyc-1` to use the zone `us-east-1-nyc-1`, US East (New York) zones.
```

Comment:
@dfitzmau still incorrect, the zone name has been added again with wrong value.

```suggestion
<1> For `<value_of_GroupName>`, specify the name of the group of the Local Zone where you want to create subnets. For example, specify `us-east-1-nyc-1` to use the zone `us-east-1-nyc-1a` (US East New York).
```

Reply:
cc @kalexand-rh (@dfitzmau mentioned he'll be on PTO the next two days)

Reply:
Updated.

Reply:
Hi @mtulio . I checked the AWS docs and I see (https://aws.amazon.com/about-aws/global-infrastructure/localzones/locations/):

![Screenshot from 2023-10-23 12-55-05](https://github.com/openshift/openshift-docs/assets/57954076/d3e97700-c79d-4d4b-af1e-0184af48e64d)

Reply:
US EAST New York > US EAST (N.Virginia)?